src/reynir_correct/pattern.py

```diff
@@ -166,6 +181,32 @@ def __init__(self, ann: List[Annotation], sent: Sentence) -> None:
                 # First instance: create the class-wide pattern list
                 self.create_patterns()
 
+    def get_wordform(self, w, cat, variants, lemma=None):
+        """ Get correct wordform from BinPackage, 
+        given a word form and a set of the variants that need to change. """
+        nouncats = frozenset(("kvk", "kk", "hk"))
```

Comment:
These sets are defined in multiple places in the code already, for instance in bintokenizer.py in GreynirPackage. Would probably be better to import one of the existing ones rather than defining a new one.